security/pkg/server/ca/server.go

```diff
@@ -164,3 +175,21 @@ func Authenticate(ctx context.Context, auth []security.Authenticator) *security.
 	serverCaLog.Warnf("Authentication failed for %v: %s", getConnectionAddress(ctx), errMsg)
 	return nil
 }
+
+// find the root cert from cert chain
+func findRootCertFromCertificateChainBytes(certBytes []byte) ([]byte, error) {
```

Comment:
can we check in the agent code if this function exists? If not, can we move this function into the security/pkg/pki package?

Reply:
moved to security/pkg/pki/util